tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -124,6 +163,18 @@ def model_to_dot(model,
     dot.set('dpi', dpi)
     dot.set_node_defaults(shape='record')
 
+  if layer_range is not None:
+    if len(layer_range) != 2:
+      raise ValueError("layer_range must be of shape (2,)")
+    if not isinstance(layer_range[0], str) or\
+      not isinstance(layer_range[1], str):
+      raise ValueError("layer_range should not contain mixed data type,",
```

Comment:
This error message should be updated to "layer_range should contain string type only."

Reply:
done